dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/CrudControllerAdvice.java

```diff
@@ -513,6 +514,18 @@ public WebMessage illegalArgumentExceptionHandler(IllegalArgumentException ex) {
     return badRequest(ex.getMessage());
   }
 
+  /**
+   * Handles {@link RuntimeJsonMappingException} and logs the stack trace to standard error. {@link
```

Comment:
```suggestion
   * Handles {@link RuntimeJsonMappingException} and logs the stack trace. {@link
```

where its logged depends on the effective log config.